docs/source/markdown/podman-run.1.md

```diff
@@ -781,6 +781,12 @@ Override the OS, defaults to hosts, of the image to be pulled. For example, `win
 Allow Podman to add entries to /etc/passwd and /etc/group when used in conjunction with the --user option.
 This is used to override the Podman provided user setup in favor of entrypoint configurations such as libnss-extrausers.
 
+#### **--passwd-entry**=*ENTRY*
+
+Customize the entry that is written to `/etc/passwd` when `--passwd` is used.
```

Comment:
I think we might want to clarify which /etc/passwd:
```suggestion
Customize the entry that is written to the `/etc/passwd` file within the container when `--passwd` is used.
```
If you take this or tweak it, please do the same in the other man page.